<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddImageWindow</class>
 <widget class="QWidget" name="AddImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>110</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Food Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="test2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>test2</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateViewer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>210 콤퓨타세탁 L</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>date view</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tagEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="placeLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Place</string>
   </property>
  </widget>
  <widget class="QLabel" name="test1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>test1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="placeEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tagLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
